tools/gather-sysinfo/gather-sysinfo.go

```diff
@@ -86,6 +110,12 @@ func makeSnapshot(cmd *cobra.Command, knitOpts *cmd.KnitOptions, opts *snapshotO
 		scratchDir = filepath.Join(scratchDir, opts.rootDir)
 	}
 
+	localPath := filepath.Join(scratchDir, machineInfoFilePath)
+	err = collectMachineinfo(knitOpts, localPath)
+	if err != nil {
+		return err
```

Comment:
and to make the code consistent with the comment, we should just do
```
if err != nil {
    log.Printf("error collecting machine info data: %v - continuing", err)
}
```

Reply:
done